modules/react/_examples/stories/examples/MediaModal.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {styled} from '@workday/canvas-kit-react/common';
+import {Modal} from '@workday/canvas-kit-react/modal';
+import {Box, Flex} from '@workday/canvas-kit-react/layout';
+import {Text} from '@workday/canvas-kit-react/text';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {spaceNumbers} from '@workday/canvas-kit-react/tokens';
+
+const Image = (elemProps: React.ImgHTMLAttributes<HTMLImageElement>) => {
+  const [loaded, setLoaded] = React.useState(false);
+
+  return (
+    <Box backgroundColor="frenchVanilla200">
+      <img
+        {...elemProps}
+        alt={elemProps.alt}
+        style={{
+          ...elemProps.style,
+          transition: 'opacity ease 200ms',
+          opacity: loaded ? 1 : 0,
+          display: 'block',
+        }}
+        onLoad={() => setLoaded(true)}
+      />
+    </Box>
+  );
+};
+
+const MediaContainer = styled('div')({
```

Comment:
```suggestion
const StyledMediaContainer = styled('div')({
```

Reply:
I feel it's better to recreate this component using `Box` and make it separated component as we have for `Image` here. It will still have understandable naming but with less emotion usage.

Reply:
I created `MediaContainer` as a separated component with `Box`